Runtime/Core/InstalledJavaVersionCheck/InstalledJavaVersionProvider.cs

```diff
@@ -57,10 +57,11 @@ public CheckInstalledJavaVersionResponse CheckInstalledJavaVersion(CheckInstalle
             }
 
             // example output: java version "1.8.0"
-            string[] outputWords = processOutput
+            var versionLine = outputMatch.Groups[0].ToString();
+            string[] outputWords = versionLine
```

Comment:
minor: Rename to something more meaningful like `versionNumberTokens`

Reply:
sure.